flytekit/clis/sdk_in_container/init.py

```diff
@@ -0,0 +1,32 @@
+import click
+from cookiecutter.main import cookiecutter
+
+
+@click.command("init")
+@click.option("--template", default="simple-example", help="cookiecutter template to be used")
```

Comment:
```suggestion
@click.option("--template", default="simple-example", help="cookiecutter template folder name to be used in the repo - https://github.com/flyteorg/flytekit-python-template.git")
```

Also eventually I think the repo itself should be parameterizable

Reply:
this idea of exposing the repo is interesting. So your expectation is that people will have their own private (or public forks) of the repo? In any case, if the need arises we can definitely go that way, but not in this PR, ok?